plugins/azure/virtualnetworks/noVPNgateways.js

```diff
@@ -0,0 +1,67 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'No VPN Gateways',
+    category: 'Virtual Networks',
+    domain: 'Network Access Control',
+    description: 'Ensures that virtual network gateways type is not VPC.',
```

Comment:
```suggestion
    description: 'Ensures that virtual network gateways type is not VPN.',
```
shouldn't it be VPN? also change description to something like 'Ensure virtual networks do not have VPN gateways configured' or 'Ensure virtual networks are not using VPN gateways'